iotics/lib/identity/validation/proof.py

```diff
@@ -40,6 +40,19 @@ def validate_proof(proof: Proof, public_base58: str):
 
 
 class DelegationValidation:
+    @staticmethod
+    def _is_valid_issuer_or_reusable_proof(deleg_proof: RegisterDelegationProof, public_key: RegisterPublicKey,
+                                           doc_id: str):
+        controller_issuer = deleg_proof.controller
+        if deleg_proof.proof_type == DelegationProofType.DID:
+            proof = Proof(controller_issuer, doc_id.encode('ascii'), deleg_proof.proof)
```

Comment:
either use .encode('ascii') everywhere or nowhere...

Reply:
I agree but in this library I would prefer to go with the minimal change. `encode('ascii')` was used before https://github.com/Iotic-Labs/iotics-identity-py/pull/17/files/b5eac7d3c2864438413c6f8d39451bb0d2d1b559#diff-54cde2149d50d32cb41f4f416f5e207dcbbcc03c3799341bb020b58847337b16L63 so I kept it